ocsp_test.go

```diff
@@ -250,7 +251,17 @@ func TestUnitValidateOCSP(t *testing.T) {
 	ocspRes.Status = ocsp.Good
 	ost = validateOCSP(ocspRes)
 	if ost.code != ocspStatusGood {
-		t.Fatalf("should have success. expected: %v, got: %v", ocspSuccess, ost.code)
+		t.Fatalf("should have success. expected: %v, got: %v", ocspStatusGood, ost.code)
+	}
+	ocspRes.Status = ocsp.Unknown
+	ost = validateOCSP(ocspRes)
+	if ost.code != ocspStatusUnknown {
+		t.Fatalf("should have failed. expected: %v, got: %v", ocspStatusUnknown, ost.code)
+	}
+	ocspRes.Status = ocsp.ServerFailed
```

Comment:
OCSP all other status test. This is taken as invalid OCSP response, which will be ignored in Fail Open mode.